tauri-app/src/lib/components/charts/APYTimeFilters.svelte

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+  import ButtonTab from '$lib/components/ButtonTab.svelte';
+  import { ButtonGroup } from 'flowbite-svelte';
+
+  const TIME_DELTA_1_MONTH = 60 * 60 * 24 * 30;
```

Comment:
does this constant not already exist somewhere in the codebase?

Reply:
exists but it is not exported, and is local to a certain file, I'll move them all to one place